modules/tinymce/src/plugins/autoresize/test/ts/browser/AutoresizePluginTest.ts

```diff
@@ -186,7 +186,7 @@ describe('browser.tinymce.plugins.autoresize.AutoresizePluginTest', () => {
     });
   });
 
-  context('TINY-9123', () => {
+  context('TINY-8872', () => {
```

Comment:
Was this ticket ID wrong?

Reply:
Yes. Neither of us could figure out why it was this number, so we updated it since we were in the area anyway.